.github/workflows/ci.yml

```diff
@@ -43,7 +43,68 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
+      ### END Rails App Dependencies ###
       - run: npm install -g yarn
       - run: yarn install --frozen-lockfile
       - name: Run CI task
         run: bundle exec rake hyacinth:ci WEBPACKER_RECOMPILE=true
+
+  ci-javascript:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10', '12']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install -g yarn
+      - run: yarn install --frozen-lockfile
+      - name: Run tests
+        run: yarn run test
+
+  ci-javascript-e2e:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10', '12']
+    steps:
+      - uses: actions/checkout@v2
+      ### BEGIN Rails App Dependencies ###
```

Comment:
FYI: This `### BEGIN Rails App Dependencies ###` (and corresponding `### END Rails App Dependencies ###`) is just here temporarily because these are an exact copy of the same steps in the earlier job.  I'd ideally like to find a way to share these common steps if we end up keeping the e2e tests (which we might not!).

Reply:
I think it would be wise to break the e2e actions out into a separate ticket/PR. But to your point about shared config: I don't see anything in the [action docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions) that suggests named references aren't supported in the YML, so that should be an option (include all of a named array of step configs, and add additional inline).

Reply:
Oh, right: There's not a good way to merge sequences in yml. Hmm. Let me try some stuff.

Reply:
I think the best we could do would be to break the reusable tasks out into a separate list and have an anchor for each one - that would be some improvement.

Reply:
And we can't do that because github actions doesn't support anchors. So that's that then!